modules/reactor/src/meshgenerators/AssemblyMeshGenerator.C

```diff
@@ -545,5 +545,8 @@ AssemblyMeshGenerator::generate()
           *(*_build_mesh), elem, rgmb_name_id_map, elem_block_name, next_block_id);
     }
   }
+
+  (*_build_mesh)->find_neighbors();
```

Comment:
any reason you did find neighbors over prepare_for_use ?

Reply:
I just chose the minimum functionality needed to get things working in Griffin. Is `prepare_for_use()` preferred over `find_neighbors`? I think `find_neighbors()` should be sufficient for this bug fix since we are manually setting block names in these `generate` functions and I don't think other functions called by `prepare_for_use` need to be called as well.

Reply:
if you want partitioning or renumbering at this point then yes, if not, not really